docs/source/dev_install.md

```diff
@@ -57,13 +59,13 @@ permissions on npm and pip related install directories are correct.
 - If troubleshooting an upgrade and its build, you may need to do the
   following process:
   
-      - Deep clean of the cloned repository:
+    - Deep clean of the cloned repository:
       
-             git clean -dfx .
+            git clean -dfx .
 
-      - Remove anything with `widgetsnbextension` in the name of files within
+    - Remove anything with `widgetsnbextension` in the name of files within
         the `conda` directory
         
-      - Remove `node_modules` directory
+    - Remove `node_modules` directory
```

Comment:
@fperez It may not be. Wasn't sure if `git clean` would nuke it. @SylvainCorlay ?